app/package.json

```diff
@@ -22,10 +22,10 @@
   "author": "Ralf Schimmel <ralf@gynzy.com>",
   "license": "MIT",
   "dependencies": {
-    "@google-cloud/bigquery": "^0.4.0",
+    "@google-cloud/bigquery": "^1.3.0",
```

Comment:
zouden we deze niet liever willen pinnen? :)

Reply:
ik zal ze allemaal wel pinnen